circle.yml

```diff
@@ -1,13 +1,24 @@
 machine:
   xcode:
-    version: "6.3.1"
+    version: "8.1.0"
+  environment:
+    FASTLANE_SKIP_UPDATE_CHECK: true
+    BUNDLE_PATH: vendor/bundle
+    UPLOAD_IOS_SNAPSHOT_BUCKET_NAME: eidolon-ci
 dependencies:
+  cache_directories:
+    - vendor/bundle
+    - ~/.cocoapods/repos
   override:
```

Comment:
@alloy I got a `pod install` down to a very small amount of time on CI now using these techniques ^=

https://circleci.com/gh/artsy/eigen/3539?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link
<img width="996" alt="screen shot 2016-11-22 at 19 28 23" src="https://cloud.githubusercontent.com/assets/49038/20538613/dfe906b2-b0e9-11e6-82a3-d0b3afacf68e.png">